src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,17 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => (
```

Comment:
Here you can destructure properties from comment

Reply:
@FaiHamid is right.
You can destructure it 
`const {name, email, body} = comment`

Reply:
@dinaminahi 

I did destructorisation, but the tests didn't work. Perhaps the tests are waiting for an object... in any case, I'll try again

Reply:
@dinaminahi @FaiHamid 
everything worked out.🫣 Thanks girls